src/validation/__tests__/harness.js

```diff
@@ -422,12 +422,20 @@ export const testSchema = new GraphQLSchema({
 });
 
 function expectValid(schema, rules, queryString) {
-  const errors = validate(schema, parse(queryString), rules);
+  const errors = validate(
+    schema,
+    parse(queryString, { experimentalFragmentVariables: true }),
```

Comment:
There are maintainers of GraphQL 3rd-party implementations that use `graphql-js` test suite as the refence so I don't think we should enable the experimental feature by default for all tests.
It's better to just add `options` as 4th argument, it would make it more explicit and also allow to easily grep for all tests cases with the particular experimental feature.